<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="searchbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지 다운로드</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>radioButton_1</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>radioButton_2</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>radioButton_3</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>radioButton_4</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>radioButton_8</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>radioButton_7</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>radioButton_6</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>radioButton_5</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>searchbox</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>radioButton_1</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>radioButton_4</tabstop>
  <tabstop>radioButton_5</tabstop>
  <tabstop>radioButton_6</tabstop>
  <tabstop>radioButton_7</tabstop>
  <tabstop>radioButton_8</tabstop>
  <tabstop>downloadButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_1</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_3</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_4</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_5</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_6</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_7</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_8</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
